packages/snjs/lib/services/search_service.ts

```diff
@@ -0,0 +1,58 @@
+import { SNNote } from "@Lib/models/app/note";
```

Comment:
This doesn't seem like a service, but more search utility functions. A service is one where we instantiate a global instance of per application, and it manages a specific task or set of operations. In this case these would be functions that the items_service would use.